03/app.js

```diff
@@ -1,3 +1,21 @@
 export default function randomNumber(min, max) {
 
-}
\ No newline at end of file
+    validateData(min, max);
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+function validateData(min, max) {
+    if( min === max) {
+        return min
+    }
```

Comment:
Nie rozumiem tej części. Skoro są takie same to zwróć mniejszą - ale dlaczego?
Tutaj ta funkcja tylko sprawdza poprawność danych, jeśli są ok to nic nie robi (choćby dlatego, że w linii 3 nie odbierasz danych zwracanych przez tą funkcje. Ten kod nic tutaj nie robi :)